tests/test_config.py

```diff
@@ -332,6 +332,10 @@ class ConfigFileTest(UsingModulesMixin, CoverageTest):
         hello = world
         ; comments still work.
         names = Jane/John/Jenny
+        
```

Comment:
Lint reports trailing space here.   There's a .editorconfig file in this repo, if you enable it in your editor, it will handle these things automatically.